DISTRIBUTION

```diff
@@ -1,11 +1 @@
 Distribution Statement A. Approved for public release. Distribution is unlimited.
```

Comment:
I put the latest DISTRIBUTION/LICENSE in here as well, to make sure the latest is available for the workshop